CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ You can find its changes [documented below](#070---2021-01-01).
 - Windows: Dark mode support for the title bar ([#2196] by [@dristic])
 - `ZStack` widget ([#2235] by [@xarvic])
 - `Lifecycle::ViewStateChanged`, `InternalLifecycle::RouteViewStateChanged`, `ChangeCtx`, and `RequestCtx` ([#2149] by [@xarvic])
+- Add a public constructor to `StringCursor` ([#2319] by [@benoitryder])
```

Comment:
Unfortunately the links aren't autogenerated. You need to manually define both `[#2319]` and `[@benoitryder]` at the bottom of the file.